appchain-registry/src/storage_key.rs

```diff
@@ -19,7 +18,6 @@ impl StorageKey {
         match self {
             StorageKey::AppchainIds => "i".to_string(),
             StorageKey::AppchainBasedatas => "a".to_string(),
-            StorageKey::ContractCode => "cc".to_string(),
             StorageKey::UpvoteDeposits => "u".to_string(),
             StorageKey::DownvoteDeposits => "d".to_string(),
             StorageKey::RegistrySettings => "rs".to_string(),
```

Comment:
We could have implemented traits: BorshStorageKey and BorshSerialize for enum StorageKey,then we can use StorageKey item as Storage prefix directly. eg from ref-exchange contract:
``` Rust
#[derive(BorshStorageKey, BorshSerialize)]
pub(crate) enum StorageKey {
    Pools,
    Accounts,
    Shares { pool_id: u32 },
    Whitelist,
    Guardian,
    AccountTokens {account_id: AccountId},
}
```

``` Rust
   Self {
            owner_id: owner_id.as_ref().clone(),
            exchange_fee,
            referral_fee,
            pools: Vector::new(StorageKey::Pools),
            accounts: LookupMap::new(StorageKey::Accounts),
            whitelisted_tokens: UnorderedSet::new(StorageKey::Whitelist),
            guardians: UnorderedSet::new(StorageKey::Guardian),
            state: RunningState::Running,
        }
```
``` rust
 tokens: UnorderedMap::new(StorageKey::AccountTokens {
                account_id: account_id.clone(),
            }),
```

Reply:
It seems the change is unnecessary. Because:

1. Current implementation is not a mistake, it is intentional. The purpose is to shorten the storage key used in actual storage operations which will reduce the over-all gas consumption in long term.
2. This change will cause all of currently using storage key changed. So we need to write a migration: read current values by old keys, write them to storage by new keys and remove them by old keys. The migration will become complicated and risky. (As current implementation works well on chain.)

I agree that the change can make the code more clean to developers, but can not bring any other real benefit to this contract.

Reply:
I totally agree,So I use `We could have` at the beginning.Maybe we can do that for the next contract.

Reply:
> It seems the change is unnecessary. Because:
> 
> 1. Current implementation is not a mistake, it is intentional. The purpose is to shorten the storage key used in actual storage operations which will reduce the over-all gas consumption in long term.
> 2. This change will cause all of currently using storage key changed. So we need to write a migration: read current values by old keys, write them to storage by new keys and remove them by old keys. The migration will become complicated and risky. (As current implementation works well on chain.)
> 
> I agree that the change can make the code more clean to developers, but can not bring any other real benefit to this contract.

I totally agree.So I used `We could have` at beginning.Maybe we can do that in next contract.

Reply:
Sure. Using macro is better in most contracts.